pkg/grizzly/providers.go

```diff
@@ -43,17 +43,22 @@ func (r Resource) String() string {
 	return r.Key()
 }
 
-// Key returns a key that combines kind and uid
-func (r *Resource) Key() string {
+func (r Resource) UID() string {
 	handler, err := Registry.GetHandler(r.Kind())
 	if err != nil {
 		return "Unknown-handler:" + r.Kind()
 	}
-	uid, err := handler.GetUID(*r)
+	uid, err := handler.GetUID(r)
 	if err != nil {
 		return "error:" + err.Error()
 	}
-	return fmt.Sprintf("%s/%s", r.Kind(), uid)
+	return uid
+}
+
+// Key returns a key that combines kind and uid
+func (r *Resource) Key() string {
+	uid := r.UID()
+	return fmt.Sprintf("%s.%s", r.Kind(), uid)
 }
```

Comment:
Could be nice to define the `.Key` method **before** the `.UID()` method, so that the file reads from higher-level methods to lower level ones. Similar to the description on [this post](https://www.reddit.com/r/ruby/comments/3ztgdo/is_there_a_proper_ruby_style_for_the_order_of/cyphi7a?utm_source=share&utm_medium=web2x&context=3).

Reply:
done